paimon-format/src/main/java/org/apache/paimon/format/orc/OrcFileFormat.java

```diff
@@ -62,13 +61,15 @@ public class OrcFileFormat extends FileFormat {
     private final Properties orcProperties;
     private final org.apache.hadoop.conf.Configuration readerConf;
     private final org.apache.hadoop.conf.Configuration writerConf;
+    private final int readBatchSize;
 
-    public OrcFileFormat(Options formatOptions) {
+    public OrcFileFormat(Options formatOptions, int readBatchSize) {
```

Comment:
I have a question? Why the `readBatchSize` attribute is not obtained from options.

Reply:
Yes, we can get `readBatchSize` by `option.get('read. batch-size')`. But I don't think we should do this way. We should use `CoreOptions#READ_BATCH_SIZE.key()`, but the `paimon-format` module does not reference the `paimon-core` module, so I pass this parameter from `paimon-core` to `paimon-format`